data/json/monsters/zed_misc.json

```
@@ -1,4 +1,85 @@
 [
+  {
+    "id": "mon_frog_mother",
+    "type": "MONSTER",
+    "name": { "str": "frog mother" },
+    "description": "A large, bulbous creature with an extremely large mouth and small, stubby legs with no feet.  It is covered in an oily greenish-black residue and large, white pustules that seem ready to pop.",
+    "default_faction": "zombie",
+    "species": [ "ZOMBIE", "AMPHIBIAN" ],
+    "volume": "240 L",
+    "weight": "300 kg",
+    "hp": 600,
+    "speed": 75,
+    "material": [ "flesh" ],
+    "symbol": "F",
+    "color": "black_green",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 1,
+    "melee_dice": 1,
+    "melee_dice_sides": 4,
+    "melee_bash": 5,
+    "dodge": 2,
+    "armor_bash": 12,
+    "armor_cut": 6,
+    "armor_bullet": 1,
+    "vision_night": 7,
+    "harvest": "zombie",
+    "special_attacks": [
```

```suggestion
    "special_attacks": [
      { "type": "leap", "cooldown": 12, "move_cost": 0, "max_range": 4, "min_consider_range": 2 },
      [ "RANGED_PULL", 10 ],
      [ "GRAB_DRAG", 3 ],
```
These describes itself as an "arm" which is too bad, but she should have something representing her ability to reach out with her sticky tongue.